code/drasil-theory/lib/Theory/Drasil/ModelKinds.hs

```diff
@@ -50,67 +50,67 @@ makeLenses ''ModelKind
 
 -- | Smart constructor for 'NewDEModel's
 newDEModel :: String -> NP -> DifferentialModel -> ModelKind e
-newDEModel u n dm = MK (NewDEModel dm) (mkUid u) n
+newDEModel u n dm = MK (NewDEModel dm) (mkUid u +++. "MK") n
```

Comment:
That the string `"MK"` occurs multiple times in this file is bad. It should occur exactly once.

Reply:
Sounds good, we can fix that (and similarly below).